README.md

```diff
@@ -10,11 +10,9 @@ FLASK_ENV=development
 
 BOARD_ID=# your board id
 
-TODO_LIST_ID=# your 'todo' list id
-DOING_LIST_ID=# your 'doing' list id
-DONE_LIST_ID=# your 'done' list id
 MONGO_USERNAME=# your Mongo DB user id
 MONGO_PASSWORD=# your Mongo DB password
+MONGO_CLUSTER=# your Mongo DB cluster
```

Comment:
Worth noting that here you've swapped the code to actually just use a single `MONGO_CONNECTION_URL`